### ISSUE TITLE
GitHub Issues migration

### ISSUE THREAD

**Issue by neilcsmith-net** [apache/netbeans#3447]
Apache NetBeans is transitioning from JIRA to GitHub issues in parallel with the release of Apache NetBeans 13.

Please open issues affecting release candidates for 13 here as we make this transition.

https://github.com/apache/netbeans/issues/new/choose

**Comment by ebresie** [comment]
May also want to consider, if an existing issue already exists in JIRA, migrate the details when raising the Github issue.  May want to comment about the issue in equivalent JIRA ticket (maybe add a link) and close out the JIRA ticket as applicable.

**Comment by neilcsmith-net** [comment]
@ebresie that will be up to the person (eg. original reporter / community member) if and when they migrate a ticket here.  There's no plan to mass migrate or link to older issues when we shutter JIRA.  We're following the process outlined at https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=191332632 fairly closely, including the last section on "How to migrate".

**Comment by SamB** [comment]
And then there are the bugzilla bugs: https://bz.apache.org/netbeans/

**Comment by ebresie** [comment]
Seems like GitHub issues has been working well. Is there any further work going on regarding this and is it ready to be closed?  If not, then what's left to do?

**Comment by asbachb** [comment]
@neilcsmith-net Can we close the issue?

**Comment by ebresie** [comment]
The summary / status of this seems to be: 

1. If there are issues still of interest in the Bugzille or JIRA issue trackers and is still a problem, then a new ticket needs to be raised  in GitHub .
2. In the new ticket transferring any relevant details (can reference old tickets as needed)
3. Once raised, then the GitHub Issues can be worked to get it resolved.

Are there still any concerns about the migration?  As there hasn't been any comments since Feb of last year I assume all is good now.  If not can this one be closed?

**Comment by Chealer** [comment]
@ebresie: It is unclear what you mean by "closed", but if Netbeans actually wants to migrate to GitHub Issues, I'm afraid quite a few features need "migration" (replication) so the data can be migrated. This GitHub tracker seems to lack both importance/priority and resolution tracking.🙁

**Comment by neilcsmith-net** [comment]
@Chealer migration is complete.  I should have closed this earlier.

**Comment by Chealer** [comment]
@neilcsmith-net: Thanks for clarifying how this was completed.

<details>
  <summary>License</summary>
This comment and any other I might add to this ticket is offered under the terms of CC0 1.0.
</details>
